.travis.yml

```diff
@@ -29,4 +29,4 @@ before_script:
     fi
 
 script:
-  - phpunit $PHPUNIT_FLAGS --exclude-group google
+  - phpunit $PHPUNIT_FLAGS --stderr --exclude-group google
```

Comment:
Why this change?

Reply:
To avoid "headers already sent" error from phpunit

Reply:
Hmm... `--stderr` redirects errors from stdout to stderr. That means there are errors printed out, right? What are these?

Reply:
Header already sent error

Reply:
Before cookie as generated after createAction is finished

Reply:
This is an error (of 7) that appear:

```
1) yii\authclient\tests\AuthActionTest::testGetDefaultSuccessUrl
session_set_cookie_params(): Cannot change session cookie parameters when headers already sent

/app/web/yii-auth-client/vendor/yiisoft/yii-web/src/Session.php:390
/app/web/yii-auth-client/vendor/yiisoft/yii-web/src/Session.php:136
/app/web/yii-auth-client/vendor/yiisoft/yii-web/src/Session.php:614
/app/web/yii-auth-client/vendor/yiisoft/yii-web/src/User.php:388
/app/web/yii-auth-client/src/AuthAction.php:167
/app/web/yii-auth-client/src/AuthAction.php:135
/app/web/yii-auth-client/tests/AuthActionTest.php:48
/app/web/yii-auth-client/vendor/phpunit/phpunit/phpunit:61
```

Reply:
Some references:

https://github.com/sebastianbergmann/phpunit/issues/1193
https://github.com/sebastianbergmann/phpunit/issues/720

Reply:
Aha. Maybe the change is not as bad as I though. @SilverFire would you please take a look? Seems you've encountered it before.

Reply:
When these changes merged, thank you

Reply:
Done.